temporal_rs/README.md

```diff
@@ -0,0 +1,38 @@
+# Temporal in Rust
```

Comment:
nit: the README.md should probably be in the top level of the crate or a different README.md should probably be made.